src/listeners/messageCreate.ts

```diff
@@ -23,5 +24,6 @@ export class MessageCreate extends Listener {
 		replyGreeting(message);
 		giveXp(message);
 		urlHelper(message);
+		answerJumble(message);
```

Comment:
to pensando depois em uma forma melhor de fazer isso é bom da load assim na mão mas acho que vai ficar meio ruim depois. Por enquanto não precisa mexer, mas a gente ja pode ir pensando em uma forma melhor de fazer isso